app/src/adapters/db/postgres_client.py

```diff
@@ -0,0 +1,146 @@
+import logging
```

Comment:
a small preference to put postgres_client in a folder `src/adapters/db/clients/postgres_client.py`, even though there's only one, just to indicate that that's where new clients should go if a project aims to define separate clients.